source/code/include/playfab/PlayFabEventApi.h

```diff
@@ -26,6 +29,8 @@ namespace PlayFab
 
         void EmitEvent(std::unique_ptr<const IPlayFabEvent> event, std::function<void(std::shared_ptr<const IPlayFabEvent>, std::shared_ptr<const IPlayFabEmitEventResponse>)> callback) const;
 
+        void Update();
```

Comment:
Update() is now only functional when the API is not threaded.   This should at least be documented someplace.